packages/samples/mvp-dapp/kuai.config.js

```diff
@@ -6,12 +6,20 @@
 
 require('dotenv').config()
 
+let redisOpt = undefined
+if (process.env.REDIS_OPT) {
+  try {
+    redisOpt = JSON.parse(process.env.REDIS_OPT)
+  } catch (error) {}
```

Comment:
Better to add a comment to indicate the error is ignored by design

Reply:
Ah it should be the same as https://github.com/ckb-js/kuai/pull/402/files#diff-7592d8066442ab2d0309fc0c5dd84caf99a8aa00a1e78e446ae0e47b9c11ad1aR3-R11